pkg/storkctl/actionstatus_test.go

```diff
@@ -0,0 +1,96 @@
+//go:build unittest
+// +build unittest
+
+package storkctl
+
+import (
+	"fmt"
+	"testing"
+
+	storkv1 "github.com/libopenstorage/stork/pkg/apis/stork/v1alpha1"
+	storkops "github.com/portworx/sched-ops/k8s/stork"
+	"github.com/stretchr/testify/require"
+)
+
+func TestGetFailoverNoFailoverAction(t *testing.T) {
+	cmdArgs := []string{"get", "failover"}
+	expected := "No resources found.\n"
+	var actionList storkv1.ActionList
+	testCommon(t, cmdArgs, &actionList, expected, false)
+}
+
+func TestGetFailoverOneSuccessfulFailoverAction(t *testing.T) {
```

Comment:
Add similar tests for failback

Reply:
I haven't added failback command yet. Will add now since failover command has been reviewed